src/crimson/os/seastore/omap_manager/btree/omap_btree_node_impl.cc

```diff
@@ -370,38 +370,44 @@ OMapInnerNode::merge_entry(
     if (donor->extent_is_below_min()) {
       DEBUGT("make_full_merge l {} r {}", oc.t, *l, *r);
       assert(entry->extent_is_below_min());
-      return l->make_full_merge(oc, r).si_then([liter=liter, riter=riter,
-                                                  l=l, r=r, oc, this] (auto &&replacement){
+      return l->make_full_merge(oc, r
+      ).si_then([liter=liter, riter=riter, l=l, r=r, oc, this](auto &&replacement)
+      {
 	LOG_PREFIX(OMapInnerNode::merge_entry);
 	DEBUGT("to update parent: {}", oc.t, *this);
-        journal_inner_update(liter, replacement->get_laddr(), maybe_get_delta_buffer());
+        journal_inner_update(liter, replacement->get_laddr(),
+	  maybe_get_delta_buffer());
```

Comment:
nit, all argument share the same indention:
```
journal_inner_update(
  liter,
  replacement->get_laddr(),
  maybe_get_delta_buffer());
```
Same below.

Reply:
all addressed.